app/interfaceactiontablehandler.py

```diff
@@ -102,14 +98,17 @@ async def get(self, request):
             employee_records = iat_employee_record_table(get_employee_info.json())
 
             job_role_json = retrieve_job_roles(get_job_roles, '')
-
+            
+            iatStat = retreiveIATstatuses()
```

Comment:
This method isn't named consistently. It should probably be `retreive_iat_statuses`.

Reply:
The variable name should probably also be `iat_stats`

Reply:
Changed to iat_stats = retreive_iat_statuses()